levelkdev/swapr-dapp
# feature: improve translations

# Summary

Fixes #1018

This PR divides translations into muitple files, improves translations keys, fixes language detection and adds new messages

Full translations outside EN are yet to be completed. Unused keys should be deleted.

# To Test

1.  Open every page
- [ ] Check if there aren't keys without message
2. Change browser language
- [ ] Check if dApp detects changes and show some broswer language translations
